frontend/src/Pages/ClassManagementForm.tsx

```diff
@@ -316,7 +316,7 @@ export default function ClassManagementForm() {
                             </label>
                         </div>
                         <div className="w-32 pt-4">
-                            <SubmitButton errorMessage={errorMessage} />
+                            <SubmitButton  errorMessage={errorMessage} label ={isNewClass ? "Create Class " : "Save Changes "}/>
```

Comment:
@JamesUnlocked Small nitpick, can you remove the extra spacing between `label` and the expression?

`<SubmitButton errorMessage={errorMessage} label={isNewClass ? "Create Class" : "Save Changes"} />`

**Also,** the submit button and cancel button are extra-wide when compared to other forms. I think that we just need to get rid of  the `w-32` class on it.